help/de/Help_ch_GUIMenus.xml

```diff
@@ -4991,14 +4976,16 @@ url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.
         öffnet sich ein weiters <guilabel>Abgleichen</guilabel> Fenster, in dem Sie Gutschrift und
         Belastung mit Ihrem Kontoauszug vergleichen können.
       </para>
+
 <!-- TODO: insert Screenshot -->
 <!-- 
       <figure>
         <title>Das <emphasis>Abgleichen</emphasis>-Fenster</title>
         
       </figure>
 -->
-       <sect3 id="Tools-reconcile-win">
+
+      <sect3 id="Tools-reconcile-win">
```

Comment:
IDs in diesem Kapitel sollten mit `GUI-`beginnen, um Konflikte mit dem Kapitel Tools zu vermeiden. Vor nderung globale suche, ob es irgendwo als ref auftaucht,